tests/unit/functions/authoriser.unitTest.ts

```diff
@@ -0,0 +1,110 @@
+import {APIGatewayTokenAuthorizerEvent, Context} from "aws-lambda";
+import {StatusCodeError} from "request-promise/errors";
+import {authorizer} from "../../../src/functions/authorizer";
+import {IncomingMessage} from "http";
+import {APIGatewayAuthorizerResult} from "aws-lambda/trigger/api-gateway-authorizer";
+import {checkSignature} from "../../../src/services/signature-check";
```

Comment:
I think a named import would reduce the number of imports for this file.

Reply:
as agreed - don't want to mess with IDE import management, so resolving with no action